tests/testthat/indention_curly_brackets/multi_line_curly_round_only-out.R

```diff
@@ -0,0 +1,9 @@
+a <- function(x) {
+  x <- c(1,
+    2 + 3,
+    sin(pi))
```

Comment:
well we have not yet attempted changing the line on which a token occurs, only the position within the line. This is to be addressed in the future.